_posts/2021-06-09-running-a-full-node.markdown

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: 'Running a Full Cardano Node'
+categories:
+- 'Getting Started'
+last_modified_at: 2021-06-09
+---
+
+As a developer the best starting point to Cardano is to get a good
+understanding of the Cardano node. This knowledge is essential whether
+you want to create Dapps, wallets, integration tools, mint custom tokens
+or operate your own stake pool.
+
+We will break it down in five basic steps:
+
+1. Set up your Linux environment
+2. Clone and build cardano-node
+3. Configure the node
+4. Running and monitoring the node
+5. Interacting with the node using cardano-cli
+
+## Background
+
+The Cardano node is a full node which contains the entire blockchain
+since its genesis and acts as the main point of contact from you as
+a client to the rest of the nodes in the network. Its main
+responsibilities are to:
+
+- Uphold the Ouroboros consensus algorithm
+- Maintain the ledger and accounting model
+- Expose integration endpoints for clients and nodes
+
+![](/img/node_cardano_components_version_main.png)
```

Comment:
This image appears to be pointing somewhere else (compared to https://learn.lovelace.academy/getting-started/running-a-full-node/#background)

Reply:
I moved it so the _posts/ folder only contains articles

Reply:
Can we change it to ![](../img/node_cardano_components_version_main.png) to have a relative path out of the _posts folder?